packages/core/src/webViews/view/WebViewScreen.tsx

```diff
@@ -0,0 +1,90 @@
+/*
+ * Axelor Business Solutions
+ *
+ * Copyright (C) 2024 Axelor (<http://axelor.com>).
+ *
+ * This program is free software: you can redistribute it and/or  modify
+ * it under the terms of the GNU Affero General Public License, version 3,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, {useCallback, useEffect, useState} from 'react';
+import {WebView} from '../../components';
+import {fetchWebViewById} from '../api.helpers';
+import {ActivityIndicator} from 'react-native';
+import {headerActionsProvider} from '../../header';
+import {createWebViewActionID} from '../display.helpers';
+import {useTranslator} from '../../i18n';
```

Comment:
Tidy up import order a bit : 

```
import React, {useCallback, useEffect, useState} from 'react';
import {ActivityIndicator} from 'react-native';
import {WebView} from '../../components';
import {headerActionsProvider} from '../../header';
import {useTranslator} from '../../i18n';
import {fetchWebViewById} from '../api.helpers';
import {createWebViewActionID} from '../display.helpers';
```
External librairies and then internal imports. For internal imports, it's easier to read when you put common tools first and then specific tools